bot/cogs/snakes.py

```diff
@@ -41,8 +65,20 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
 
-    # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+        data = await self.get_snek(name)
+        await ctx.send(data)
+
+    # The snake moult command
 
+    @command()
+    async def moult(self, ctx: Context):
```

Comment:
I would prefer if this just got the bot role and changed its colour, but this is a fun command either way!